ALPINE.txt

```diff
@@ -1 +1 @@
-3.16
\ No newline at end of file
+3.18
```

Comment:
This just determines how our build infra will tag the new base and what base containerboot/operator images should use, it's ran via `./tool/go run ./cmd/printdep -alpine`